test/stripe/api_resource_test.rb

```diff
@@ -318,34 +318,32 @@ class NestedTestAPIResource < APIResource
         stub_request(:post, "#{Stripe.api_base}/v1/customers/cus_123")
           .with(body: { "description" => "another_mn" })
           .to_return(body: JSON.generate(customer_fixture))
-        c = Stripe::Customer.construct_from(customer_fixture)
-        c.description = "another_mn"
-        c.save
+        Stripe::Customer.construct_from(customer_fixture)
+        Stripe::Customer.update("cus_123", { description: "another_mn" })
       end
 
       should "updating should merge in returned properties" do
```

Comment:
I'm not sure "merge in returned properties" describes what Update does. Maybe we should just delete these tests or keep them the way they are because even though it is deprecated it is good to have test coverage for `.save` until we entirely remove it